packages/image_picker/image_picker_ios/pigeons/messages.dart

```diff
@@ -45,4 +45,10 @@ abstract class ImagePickerApi {
   @async
   @ObjCSelector('pickVideoWithSource:maxDuration:')
   String? pickVideo(SourceSpecification source, int? maxDurationSeconds);
+
+  /// Selects images and videos and returns their paths.
+  @async
+  @ObjCSelector('pickMediaWithMaxSize:quality:fullMetadata:allowMultiple:')
+  List<String?> pickMedia(MaxSize maxSize, int? imageQuality,
+      bool requestFullMetadata, bool allowMultiple);
```

Comment:
Optional: consider an options class. Because we currently don't support named params in the generated Dart code, having multiple params of the same type doesn't give a great calling experience; someone could trivially switch the bool order at the call site.

Reply:
Seems like a useful pigeon feature